zetaclient/orchestrator/bootstrap.go

```diff
@@ -481,6 +479,20 @@ func syncObserverMap(
 	return added, removed, nil
 }
 
+func btcDatabaseFileName(chain chains.Chain) string {
+	// legacyBTCDatabaseFilename is the Bitcoin database file name now used in mainnet and testnet3
+	// so we keep using it here for backward compatibility
+	const legacyBTCDatabaseFilename = "btc_chain_client"
+
+	// For additional bitcoin networks, we use the chain name as the database file name
+	switch chain.ChainId {
+	case chains.BitcoinMainnet.ChainId, chains.BitcoinTestnet.ChainId:
+		return legacyBTCDatabaseFilename
+	default:
+		return fmt.Sprintf("%s_%d", legacyBTCDatabaseFilename, chain.ChainId)
```

Comment:
we use names of chains rather than chain ID currently so it would be good to keep that consistent?

```
zetachain@validator1-us-east-1-testnet:~/.zetaclient/chainobserver$ ls
amoy_testnet  base_sepolia  bsc_testnet  btc_chain_client  goerli_testnet  mumbai_testnet  scanner	sepolia_testnet  solana_devnet
```

Reply:
I don't have preference but using name as suffix also makes sense to me: https://github.com/zeta-chain/node/pull/3225/commits/7b113bd20aa6699e80946d92c82f6cdaa4c3b15b